base/ltboxes.dtx

```diff
@@ -376,15 +381,38 @@
         \expandafter\let\csname mb@\reserved@a\endcsname\relax
       \fi}%
     \mb@t
-    \hb@xt@ #1\unitlength{\mb@l #4\mb@r}%
+    \@defaultunitsset\@tempdimc{#1}\unitlength
+    \hb@xt@\@tempdimc{\mb@l #4\mb@r}%
     \mb@b
+    \kern\z@}}
+%</2ekernel|latexrelease>
+%    \end{macrocode}
+%    
+%    \begin{macrocode}
+%<latexrelease>\EndIncludeInRelease
+%<latexrelease>\IncludeInRelease{0000/00/00}%
+%<latexrelease>                 {\@imakepicbox}{default units}%
+%<latexrelease>\long\def\@imakepicbox(#1,#2)[#3]#4{%
+%<latexrelease>  \vbox to#2\unitlength
+%<latexrelease>   {\let\mb@b\vss \let\mb@l\hss\let\mb@r\hss
+%<latexrelease>    \let\mb@t\vss
+%<latexrelease>    \@tfor\reserved@a :=#3\do{%
+%<latexrelease>      \if s\reserved@a
+%<latexrelease>        \let\mb@l\relax\let\mb@r\relax
+%<latexrelease>      \else
+%<latexrelease>        \expandafter\let\csname mb@\reserved@a\endcsname\relax
+%<latexrelease>      \fi}%
+%<latexrelease>    \mb@t
+%<latexrelease>    \hb@xt@ #1\unitlength{\mb@l #4\mb@r}%
+%<latexrelease>    \mb@b
+%<latexrelease>    \kern\z@}}
+%<latexrelease>\EndIncludeInRelease
+%<*2ekernel>
```

Comment:
I would keep the comment at the place where it belongs, i.e. 3 line above

Reply:
I moved it to above the matching kern in the current code not the latexrelease block.